people.rkt

```diff
@@ -242,6 +242,21 @@
                   #:img "steven_holtzen.png"]{
             My research focuses on programming languages and artificial intelligence. In particular, my goal is to use programming languages and program analysis techniques as a foundation for specifying and reasoning about probabilistic models. Towards this end I am interested in the design, implementation, and applications of probabilistic programming languages; foundations of probabilistic inference and tractable probabilistic modeling; automated reasoning; and probabilistic verification.
           }
+
+          @person[#:name "Daniel Patterson"
+                  #:title "Assistant Teaching Professor"
+                  #:e-mail "dbp@ccs.neu.edu"
```

Comment:
Could you actually switch this to dbp@dbpmail.net ? I can't send as it, so it's somewhat annoying to receive mail to. (and somewhat out of principle, don't really want to put @northeastern.edu one)

Reply:
Done